prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -7371,6 +7389,8 @@ subroutine getParserVersion(node, versionString, parserVersion)
     integer, intent(out) :: parserVersion
 
     select case(trim(versionString))
+    case("20.2")
+      parserVersion = 9
```

Comment:
This should be done with the release not with this PR, I think. Goes back to the argument in https://github.com/dftbplus/dftbplus/issues/592#issuecomment-667562199, DFTB+ will still print 20.1 as version, I think the input file should not bypass this.

Reply:
@awvwgk hopefully resolved